### ISSUE TITLE
Dependency issue

### ISSUE THREAD

**Issue by pcriadoperez** [NervanaSystems/coach#300]
Installation is failing with this dependency issue:

```
requests 2.18.4 has requirement urllib3<1.23,>=1.21.1, but you'll have urllib3 1.25 which is incompatible.
kubernetes 8.0.1 has requirement urllib3>=1.23, but you have urllib3 1.22.
```

**Comment by galleibo-intel** [comment]
I get a slightly modified version of this error - 
```ERROR: requests 2.21.0 has requirement urllib3<1.25,>=1.21.1, but you'll have urllib3 1.25 which is incompatible.```

It can be work-arounded by editing requirements.txt and freezing urllib3 to version 1.24.2, by adding the following line - 
```urllib3==1.24.2```

requests is required by gym, and urllib3 >= 1.23 is requested by kubernetes.

**Comment by galnov** [comment]
This was fixed on requests v2.22.0, so no failure now.
